app/controllers/dishes_controller.rb

```diff
@@ -0,0 +1,6 @@
+class DishesController < ApplicationController
+  def show
+    @dish = Dish.find(params[:id])
+    @ingredients = @dish.ingredients
```

Comment:
Same idea here -- you can always call @dish.ingredients in the view -- that wouldn't be an MVC infraction because you are just calling a model association rather than using any AR querying methods